### PR TITLE
texi2html: update to 5.0.

### PR DESCRIPTION
**Pull request by DirectorX**

### CODE REVIEW

**Comment by newbluemoon** [comment]
The reason I didn’t update this one is that I initially needed it to build the documentation for lilypond and this doesn’t work with later versions than 1.82 (there’s an API change). However, lilypond uses the pre-built documentation because of some cross-compilation problems, anyway, so I think the update is ok. I’m not aware of any other software having problems with 5.0.

File: srcpkgs/texi2html/template [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Template file for 'texi2html'
 pkgname="texi2html"
```
Comment (maxice8) [inline srcpkgs/texi2html/template]: please unquote this

File: srcpkgs/texi2html/template [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
-version="1.82"
+version="5.0"
 revision=1
 noarch=yes
 build_style="gnu-configure"
```
Comment (maxice8) [inline srcpkgs/texi2html/template]: unquote this too

File: srcpkgs/texi2html/template (hunk 1 above)
Comment (DirectorX) [reply srcpkgs/texi2html/template]: done

File: srcpkgs/texi2html/template (hunk 2 above)
Comment (DirectorX) [reply srcpkgs/texi2html/template]: done

**Pull request merged**